### DISCUSSION TITLE
Nuova installazione Home Assistant + BTicino

### DISCUSSION THREAD

**Discussion by Nosari89** [indomus/forum#2836] [HUB: Home Assistant]
Ciao a tutti ragazzi, spero di essere nella sezione corretta e spero che qualcuno possa darmi qualche consiglio
sto approcciando ora il mondo HA, sto seguendo varie guide e tutorial per muovere i primi passi; sto piano piano facendo casa nuova e vi vorrei esporre i miei dubbi principali:

### 1) INSTALLAZIONE DI HA
al momento per provare a giocare ho due istanze di HA:
- una su Synology DS218, che purtroppo non può gestire VM o Docker, sono solo riuscito ad installare dalla community HA Core; su InDomus vedo che è il più consigliato, ma anche il più da "smanettoni" e finché non sarò pratico penso non sia la mia scelta corretta
- una sul mio pc personale con HAOS su Virtualbox e per ora direi che va tutto bene

detto questo sto pensando a quale sia l'installazione definitiva per la casa nuova e stavo pensando ad un mini pc, come il Geekom mini Air 12, per i bassi consumi dato che resterà acceso H24 e dato che non dovrò farci girare molta roba
preferirei tenere il mini pc con Win11 e tenere HA su Virtualbox (al posto che flashare tutto l'SSD con HA perché direi proprio che è eccessivo [16 Gb Ram - 512 Gb SSD]) e soprattutto perché magari in un futuro vorrò farci girare anche qualcos'altro.... non lo so, un DNS, una VPN, Plex server, ecc... tutte cose che farebbe già anche il mio Synology lo so... su quello ci ragionerò =)

dite che è una soluzione fattibile/sensata ?

### 2) FREQUENZA DI AGGIORNAMENTO DISPOSITIVI 
da alcune prove e letture sui forum noto che alcuni dispositivi, dopo per esempio un comando di accensione/spegnimento, ci mettono molto ad aggiornare lo stato su HA, esiste un parametro per aumentare la "frequenza di controllo", penso che ovviamente sia frustrante avere tutto smart e una cosa che va molto lenta

### 3) GEOFENCING
per la rilevazione GPS di una persona in una zona, esempio il mio cellulare con app HA e la zona CASA, c'è un modo per impostare la zona CASA con frequenza di aggiornamento posizione diciamo "altissima" e impostare una zona più grande circondariale a CASA con frequenza di aggiornamento più lenta ?
mi spiego, per non drenare costantemente la batteria del telefono (ho un iPhone), posso aggiornare la posizione, non lo so, una volta ogni 5 min se sono molto lontano...... come entro nella zona circondariale ogni 1 min.... come entro nella zona CASA ogni 30 sec ?
è un esempio a caso ma è solo per rendere l'idea, così da non dover mandare mille aggiornamenti GPS anche se non servono

**PS: ma come si cambia il raggio della zona HOME una volta che è impostata ? =)**

### 4) INTEGRAZIONE BTICINO LIVING LIGHT SMART
arriviamo alla parte più ostica, nella casa nuova ho fatto tutto con l'ecosistema BTicino composto da:
- 1 gateway FC80GT
- 14 deviatori connessi L4003C
- 7 prese connesse L4531C
- 1 gestore carichi F20T60A
- 1 contattore FC80CC
- 1 misuratore F20T60A
sto leggendo ovunque che ci sono un sacco di problemi, o dispositivi che non si integrano, o che non si aggiornano, o che posso fare on/off ma non leggere i consumi, ma soprattutto mi pare di aver capito che se integrassi tutto con HA tramite zigbee perderei l'app Home+ Control, e sinceramente mi rugherebbe parecchio perdere un'app che penso funzioni bene per integrarlo in un sistema che ha problemi, anche sinceramente per il costo sostenuto
c'è qualcuno che ha risolto questa mia stessa situazione ?

per zigbee l'idea era di usare Z2M con una Sonoff ZBDongle-E v2

**PS: vedo che molti parlano di integrazione tramite HomeKit, senza entrare nel merito, ma come fate ad avere l'ecosistema BTicino su home kit da remoto ? sono l'unico st....o che non ci riesce ? =)
a me funziona tutto se sono in LAN, come esco non va niente (su Home kit)**

ho visto in qualche post la soluzione "sdomotica" che è a pagamento e su Cloud, qualcuno ha qualche dettaglio in più (costi/benefici)?
so che per gli amanti di HA non bisognerebbe dipendere dai cloud, ma (commento da ignorante) se cadesse internet comando comunque tutte le luci a tasti fisici per quel momento in cui manca la connessione volendo, ci sono altri "disagi" nel dipendere da un cloud ?

stessa cosa vale per l'integrazione di tutto l'ecosistema Alexa, vale la pena pagare HA Cloud o ci sono vie migliori ?
anche qui.... costi/benefici ?

Chiedo scusa fin da subito per il pippone =)
spero che qualche anima pia possa sbrogliare i miei dubbi senza entrare troppo nel tecnico in quanto principiante

Grazie a tutti per un qualsiasi consiglio :)

**Comment by VincenzoMaletta** [comment]
Ciao Nosari89,

L'integrazione di LivingLight con HA può seguire diverse strade. La scelta principale da fare è riguardo l'avere la gestione dei sispositivi completamente in locale oppure attraverso il cloud. Nel primo caso l'unica soluzione è l'utilizzo di Z2M o similari, perdendo la possibilità di usare l'app Home + Control. Nel secondo caso puoi scegliere se connettere HA tramite Homekit (anche se non usi l'ecosistema Apple) o NetAtmo. Funzionano ambedue ugualmente bene e ti consentono di controllare i dispositivi tramite HA o app proprietaria. Hanno dei limiti, nel senso che non consentono il controllo completo dei dispositivi; ad esempio non riportano i consumi delle prese controllate. Se cerchi in rete trovi diverso materiale.

Ne approfitto per chiederti una cosa. Sto iniziando ad esplorare lecosistema LivingLight ed ho cominciato a "giocare" con un gateway ed un L4003C. Il controllo del dispositivo dall'app o da HA funziona correttamente mentre il controllo manuale non aggiorna lo stato del dispositivo, né ad HA né all'app. Non avendo altri dispositivi da confrontare non so se sia un comportamento normale o se sia difettoso il mio interruttore.

Grazie anticipato per un eventuale supporto.

**Reply by Nosari89** [reply]
Ciao Vincenzo,

Per quanto riguarda il tuo caso sinceramente mi cogli un po' impreparato, perché io sto facendo tutto in una casa nuova e, non essendo sempre lì, forse non ci ho mai fatto caso
Di sicuro ti posso assicurare che se accendo delle luci manualmente sull'app poi le vedo accese, non so però dirti se la cosa è istantanea e/o serve un aggiornamento, non ci ho mai fatto caso
Ma se apro l'app con delle luci accese, fidati che dall'app le vedo accese

Per quanto riguarda invece il mio caso, ho dei "contro" in entrambi i metodi
- se uso Z2M perdo l'app proprietaria
- se uso il cloud non leggo i consumi, scelta già da bocciare direi
In rete si parla di Sdomotica, però oltre al costo e l'installazione un po' complicata, serve o un Raspberry separato o avere tutto HA già su un raspberry (io volevo mettere HA in una virtual machine su mini pc)
E mi pare che si possono fare solo un TOT di "interrogazioni" al giorno sui dispositivi, quindi non so...

In più ho un'altro dilemma
Io sono al secondo piano e ho il garage al -1
In garage potrei portare internet con un powerline, ma se metto qualche dispositivo zigbee come faccio ?
Dato che la chiavetta zigbee dev'essere collegata via usb al pc ? ☺️

**Reply by Scialla** [reply]
per usare Homekit da remoto devi usare un hub Apple, una AppleTV o un HomePod

l'aggiornamento di stato dei dispositivi è istantaneo, se tu hai delle latenze devi indagare sui log, su tutti i log interessati dal passaggio delle informazioni del device; ad es. se è zigbee, sui log di z2m/zha, di mqtt, di HA

la frequenza di aggiornamento della posizione degli smartphone, si imposta da smartphone ma, solitamente, puoi impostare poco, ad es. normale/accurata
